jdrfandroidbleparser/src/main/java/org/ehealthinnovation/jdrfandroidbleparser/encodedvalue/dis/bgmmeasurement/SampleLocation.kt

```diff
@@ -0,0 +1,18 @@
+package org.ehealthinnovation.jdrfandroidbleparser.bgm.encodedvalue.bgmmeasurement
+
+import java.nio.channels.spi.AbstractSelectionKey
+
+enum class SampleLocation constructor(val key: Int) {
+    RESERVED_FOR_FUTURE_USE(0),
+    FINGER(1),
+    ALTERNATIVE_SITE_TEST(2),
+    EARLOBE(3),
+    CONTROL_SOLUTION(4),
+    SUBCUTANEOUS_TISSUE(5),
```

Comment:
I'm assuming this is new, as it is not on the website?

Reply:
This value key exists for he CGM case. Have removed it from the Glucose Profile